CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Coding guidelines
```

Comment:
I'm aware of that.

But my bar for a file that gets that link is higher than this file currently meets. For example: https://github.com/rust-lang/rust/blob/master/CONTRIBUTING.md

That file has the following sub-sections:

## Feature Requests
## Bug Reports
## The Build System
## Pull Requests
## Writing Documentation
## Issue Triage
## Out-of-tree Contributions
## Helpful Links and Information

That's sort of what I expect when I click the bar that GitHub helpfully shows. I don't expect "here's three gotchas about QStringLiteral".